PurchasesTests/Purchasing/OfferingsManagerTests.swift

```diff
@@ -43,43 +43,43 @@ class OfferingsManagerTests: XCTestCase {
 
 extension OfferingsManagerTests {
 
-    func testOfferingsForAppUserIDReturnsNilIfMissingProductDetails() {
+    func testOfferingsForAppUserIDReturnsNilIfMissingProductDetails() throws {
```

Comment:
for some of these I updated to use `XCTUnwrap`.
As a reminder, the advantage to using `XCTUnwrap` + `throws` vs implicitly-unwrapped optionals (i.e.: `!`) is that if the object is nil, an implicitly-unwrapped optional would crash the entire test suite and stop tests right there, but `XCTUnwrap` would just mark the test as failed and continue to test the remaining test cases.

Reply:
This is a great sharpen the saw task, or if somebody wants to help us out! https://github.com/RevenueCat/purchases-ios/issues/881